lib/dockly/docker.rb

```diff
@@ -33,12 +33,27 @@ def build_env(hash = nil)
   end
 
   def generate!
-    image = generate_build
+    image = find_image_by_repotag || generate_build
```

Comment:
Actually, this can be removed here.  This is only called by dockly itself.  Not from the rake tasks.